pkg/controller/configmap/configmap_controller.go

```diff
@@ -196,6 +196,13 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 			}
 		}
 	}
+    networkConfigChanged := true
```

Comment:
is it simpler if we can do:
`networkConfigChanged := hasNetworkConfigChanged(networkConfig, r.sharedInfo.NetworkConfig)`

Reply:
makes sense. I think I had something different in mind initially!

Reply:
Ah I did set it to trye here to ensure it was defined even if NetworkConfig was not yet populated (first reconcilation)

Reply:
indent seems ok on last PR update